src/table/cells/handle-cell-focus.js

```diff
@@ -0,0 +1,25 @@
+export const focusCell = (rowElements, nextCellCoord) => {
+  const nextCell = rowElements[nextCellCoord[0]].getElementsByClassName('sn-table-cell')[nextCellCoord[1]];
+  nextCell.focus();
+  nextCell.setAttribute('tabIndex', '0');
+};
+
+const removeTabSequence = (rowElements, nextCellCoord) => {
+  const cell = rowElements[nextCellCoord[0]]?.getElementsByClassName('sn-table-cell')[nextCellCoord[1]];
+  cell && cell.setAttribute('tabIndex', '-1');
+};
+
+export const handleCellFocus = (cell, focusedCell, setFocusedCell, rootElement) => {
+  const { rawRowIdx, rawColIdx } = cell;
+
+  removeTabSequence(
+    rootElement.getElementsByClassName('sn-table-row'),
```

Comment:
This is fetched twice, better to do it once and save it as a `const`

Reply:
good point. gonna fix.